action.yml

```diff
@@ -18,6 +18,10 @@ inputs:
     type: boolean
     required: true
     default: false
+  version:
+    description: 'Which version of the `cargo semver-checks tool to install.'
```

Comment:
```suggestion
    description: 'Which version of the `cargo-semver-checks` tool to install.'
```

Reply:
Would you mind explicitly documenting how to pass a version here with an example? Right now it seems to be asking for a version but actually requires a tag name complete with the leading `v`.

Or perhaps it might be best to make the action logic automatically prepend a `v` if the value passed here instead starts with just a number?